jgrapht-ext/src/main/java/org/jgrapht/ext/DOTImporter.java

```diff
@@ -90,6 +90,12 @@
 public class DOTImporter<V, E>
     implements GraphImporter<V, E>
 {
+
+    /**
+     * Default key for being used in the graph updater (if provided) to add the graph ID.
```

Comment:
A bit picky here, but could you fix the comment here? Something like "Default key used in the graph updater (if provided) for the graph ID".